api/src/AppBundle/v2/Fixture/Controller/FixtureController.php

```diff
@@ -66,7 +68,13 @@ public function createCourtOrderAction(Request $request)
 
         $client = $this->createClient($fromRequest);
         $deputy = $this->createDeputy($fromRequest);
-        $this->createReport($fromRequest, $client);
+
+        if (strtolower($fromRequest['reportType']) == 'ndr') {
```

Comment:
Fussy, but I have a twitch about this from JavaScript:
```suggestion
        if (strtolower($fromRequest['reportType']) === 'ndr') {
```